torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -373,6 +374,11 @@ class FullyShardedDataParallel(nn.Module):
             avoid sharding specific parameters when using an
             ``auto_wrap_policy`` or if parameters' sharding is not managed by
             FSDP. (Default: ``None``)
+        device_id (Optional[int]): An `int` describing the CUDA device FSDP
```

Comment:
nit: I think in the docstring, we need double backtick to render as code, so \'int\' -> \`\`int\`\`.